packages/react-ui-components/src/Tree/node.js

```diff
@@ -222,16 +224,17 @@ export class Header extends PureComponent {
                                         <IconComponent icon={icon || 'question'} label={iconLabel} />
                                     }
                                 </div>
-                                <span
+                                <a
                                     {...rest}
                                     id={labelIdentifier}
                                     className={theme.header__label}
-                                    onClick={onLabelClick}
+                                    onClick={onLabelClick ? onLabelClick : (directLink ? this.onDirectLinkClick : undefined)}
```

Comment:
this line is a bit hard to read ... 

just a thought. what would you think about:
```js
const linkHandlingProps = directLink
  ? {
  href: directLink,
  onClick: (e) => e.preventDefault()
  }
  : {
  onClick: onLabelClick
  };

return <a
 other="props"
 {...linkHandlingProps}
/>

```

Reply:
@markusguenther ping pong